src/site/markdown/getting-started.md

```diff
@@ -17,58 +17,66 @@
 
 ## Getting Started
 
-To on-board incremental Maven you need to complete several steps:
+To onboard incremental Maven you need to complete several steps:
 
 * Declare caching extension in your project (either in `pom.xml` or `.mvn/extensions.xml`)
-* Add `maven-build-cache-config.xml` cache config in `.mvn/` (optional) to customize default behavior
-* Validate build results and iteratively, adjust config to properly reflect project specifics
+* Add `maven-build-cache-config.xml` cache config in `.mvn/` (optional) to customize the default behavior
+* Validate build results and iteratively adjust config to reflect project specifics properly
 * Setup remote cache (optional)
 
-### Declaring build cache extension
+### Declaring build-cache extension
 
 ```xml
+
 <extension>
     <groupId>org.apache.maven.extensions</groupId>
     <artifactId>maven-build-cache-extension</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0</version>
```

Comment:
we should maybe use some interpolation here.

Reply:
I will put the `LATEST` for simplicity.